UnityMcpBridge/UnityMcpServer~/src/tools/resource_tools.py

```
@@ -132,26 +136,16 @@ def _resolve_safe_path_from_uri(uri: str, project: Path) -> Path | None:
-        pattern: Optional[str] = "*.cs",
-        under: str = "Assets",
-        limit: Any = 200,
-        project_root: Optional[str] = None,
-    ) -> Dict[str, Any]:
-        """
-        Lists project URIs (unity://path/...) under a folder (default: Assets).
-        - pattern: glob like *.cs or *.shader (None to list all files)
-        - under: relative folder under project root
-        - limit: max results
-        """
+        ctx: Context,
```

Ruff (0.13.1)

Unused function argument: `ctx`

(ARG001)